app/Http/Resources/V1/ProjectDetailedResource.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Resources\V1;
+
+use App\Http\Resources\V1\ProjectUserResource;
+use App\Http\Resources\V1\ProjectPlaylistResource;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class ProjectDetailedResource extends JsonResource
```

Comment:
Maybe, it makes more sense to make this ProjectDetailResource.
Update references to this class accordingly.

Reply:
done